packages/docusaurus-plugin-content-docs/src/sidebars.ts

```diff
@@ -97,7 +97,7 @@ function assertItem<K extends string>(
 function assertIsCategory(
   item: unknown,
 ): asserts item is SidebarItemCategoryJSON {
-  assertItem(item, ['items', 'label', 'collapsed', 'customProps']);
+  assertItem(item, ['items', 'label', 'collapsed', 'link', 'customProps']);
```

Comment:
We need to validate that the link attribute is valid in this function + add unit tests

Reply:
I might need some help on this one